tests/utils/misp-ioc-sender/README.md

```diff
@@ -0,0 +1,36 @@
+# MISP IoC Sender
+
+A script to trigger an Attribute update in MISP. The script is configured via a
+YAML file. See belog for details and check out the `config.yaml.example` file.
+
+The script works as follows:
+
+- It looks for a specific MISP Event, referenced by its UUID in the config file.
+- The script creates the Event if it doesn't exist yet.
+- The script checks the Attributes of the Event and either creates a new
+  Attribute or it toggles the `'to_ids'` flag if the Attribute already exists.
+
+Attributes created by this script always have the `"DOMAIN"` type. Attribute
```

Comment:
Why always `DOMAIN`?

Reply:
That's what we agreed to use with DCSO for testing. We can make this configurable, if you'd like.

Reply:
Let's keep it as is and update only if we need the extra flexibility.